services/brig/test/integration/API/Federation.hs

```diff
@@ -41,9 +42,17 @@ testGetUserByHandleSuccess brig = do
   hdl <- randomHandle
   putHandle brig uid hdl
     !!! const 200 === statusCode
-  get (brig . paths ["federation", "users", "by-handle"] . queryItem "handle" (toByteString' hdl)) !!! do
-    const 200 === statusCode
-    const (Just (UserHandleInfo quid)) === (responseJsonMaybe)
+  profile <-
```

Comment:
It's nice to change the test when changing the API; however, the previous expectation should still hold for backwards compatibility, and parsing the response as UserHandleInfo should still work here. Could you test that the response parses as both the old and the new type if you'd like to change that endpoint?

Reply:
Yes, this is a breaking change to the federation API and I think it is fine to make this in our current state. Is it not?

Reply:
> shouldn't be used by anyone yet, so it is safe to break.

I couldn't find any recent usage of that in the logs, so maybe it's not in use yet indeed.

Still, the userhandle object gives `qualified_user` whereas the userprofile gives `qualified_id`. It's not consistent and it's not clear to me why we can't be consistent in the naming of these things.